test/e2e/framework/framework.go

```diff
@@ -56,7 +56,9 @@ import (
 const (
 	// DefaultNamespaceDeletionTimeout is timeout duration for waiting for a namespace deletion.
 	DefaultNamespaceDeletionTimeout = 5 * time.Minute
-	defaultServiceAccountName       = "default"
+	// DefaultObservationTimeout is duration where checking kubernetes object status.
```

Comment:
There might be a little typo in `is duration where checking`, maybe this should mean something along the lines of
`// DefaultObservationTimeout specifies the maximum duration to wait when checking the status of kubernetes objects`?